boards/imixv2/Cargo.lock

```diff
@@ -0,0 +1,50 @@
+[root]
+name = "imix"
```

Comment:
Is the plan to make this look like imix and replace what is now the imix board ASAP? Or should all of these change to "imixv2"? I can see an argument for moving the current imix to imixv1 and making this just imix.